wander-embedded-sdk/README.md

```diff
@@ -29,6 +29,8 @@
 
   useEffect(() => {
     const wanderInstance = new WanderEmbedded({
+      applicationId: "<applicationId>",
+      clientId: "<clientId>",
```

Comment:
Because client ID and application ID map 1-to-1, I think they should only provide the client ID (just like with Auth0, for example). With that, the verification endpoint can load the application already.